cisco_aci/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - cisco_aci
 
+## 2.2.1 / 2022-12-09
+
+* [Fixed] Update Snowflake connector and cryptography dependencies. See [#13367](https://github.com/DataDog/integrations-core/pull/13367).
```

Comment:
```suggestion
* [Fixed] Update cryptography dependency. See [#13367](https://github.com/DataDog/integrations-core/pull/13367).
```

The snowflake connector and cryptography updates happened on the same PR, but not all are affected by the snowflake-connector update.